site/content/docs/5.1/components/navs-tabs.md

```diff
@@ -335,44 +335,54 @@ Use the tab JavaScript plugin—include it individually or through the compiled
 <div class="bd-example">
   <ul class="nav nav-tabs mb-3" id="myTab" role="tablist">
     <li class="nav-item" role="presentation">
-      <button class="nav-link active" id="home-tab" data-bs-toggle="tab" data-bs-target="#home" type="button" role="tab" aria-controls="home" aria-selected="true">Home</button>
+      <button class="nav-link active" id="home-tab" data-bs-toggle="tab" data-bs-target="#home-tab-pane" type="button" role="tab" aria-controls="home-tab-pane" aria-selected="true">Home</button>
     </li>
     <li class="nav-item" role="presentation">
-      <button class="nav-link" id="profile-tab" data-bs-toggle="tab" data-bs-target="#profile" type="button" role="tab" aria-controls="profile" aria-selected="false">Profile</button>
+      <button class="nav-link" id="profile-tab" data-bs-toggle="tab" data-bs-target="#profile-tab-pane" type="button" role="tab" aria-controls="profile-tab-pane" aria-selected="false">Profile</button>
     </li>
     <li class="nav-item" role="presentation">
-      <button class="nav-link" id="contact-tab" data-bs-toggle="tab" data-bs-target="#contact" type="button" role="tab" aria-controls="contact" aria-selected="false">Contact</button>
+      <button class="nav-link" id="contact-tab" data-bs-toggle="tab" data-bs-target="#contact-tab-pane" type="button" role="tab" aria-controls="contact-tab-pane" aria-selected="false">Contact</button>
+    </li>
+    <li class="nav-item" role="presentation">
+      <button class="nav-link disabled" id="disabled-tab" data-bs-toggle="tab" data-bs-target="#disabled-tab-pane" type="button" role="tab" aria-controls="disabled-tab-pane" aria-selected="false" disabled>Disabled</button>
```

Comment:
Yes. The current rule is:
```scss
&.disabled {
    color: var(--#{$prefix}nav-link-disabled-color);
    background-color: transparent;
    border-color: transparent;
}
```
We could add `:disabled` as a selector as well for example. But I'm waiting for Patrick's suggestion.

Reply:
+1 to using actual `disabled` attribute on `<button>`, but also allowing the same to happen with `.disabled` when author chose to use link or something

Reply:
Thanks for the feedback Patrick. Following the discussion in this comment, https://github.com/twbs/bootstrap/pull/36169/commits/d613fa0b80c6c37dccbdc4c3eaeed1f5faeff331 uses `disabled` attribute on `<button>` instead of `.disabled`. CSS rules have been added in `_nav.scss` to cover this use case.